### DISCUSSION TITLE
[Question] Is it possible that the ViewRenderable only renders in front of the marker?

### DISCUSSION THREAD

**Discussion by paramadharmika** [sceneview/sceneview#303] [Q&A]
I'm trying to render an image after a successful scan. So far I'm able to do that with ViewRenderable,  it shows in front of the marker (the plane is vertical) but it also shows on the back side of the marker. 

Is it possible to show ViewRenderable only in front of the marker? If not, what would the alternative be? Maybe render it as texture?

I don’t know much about this topic, I'd appreciate it if anyone could shed some light in the right direction

Thank you

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, `ViewNode` renders in 3D space and respects depth ordering by default. It will be occluded by objects in front of it. If you want a View to always render on top (like a HUD/label), you'd need to adjust the render priority or disable depth testing on that node's material.

For AR labels that should always face the camera, set the node's rotation to track the camera direction each frame in `onFrame`.
